src/backend/marsha/bbb/migrations/0007_classroom_rename_meeting_to_classroom.py

```diff
@@ -0,0 +1,108 @@
+        ),
+        migrations.RemoveConstraint(
+            model_name="classroom",
+            name="meeting_unique_idx",
+        ),
+        migrations.AlterField(
+            model_name="classroom",
+            name="meeting_id",
+            field=models.UUIDField(
+                default=uuid.uuid4,
+                editable=False,
+                help_text="BBB meeting id as UUID",
+                verbose_name="BBB meeting id",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="classroom",
+            name="description",
+            field=models.TextField(
+                blank=True,
+                help_text="description of the classroom",
+                null=True,
+                verbose_name="description",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="classroom",
+            name="estimated_duration",
+            field=models.DurationField(
+                blank=True,
+                help_text="Estimated duration of the classroom in seconds.",
+                null=True,
+                verbose_name="estimated duration",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="classroom",
+            name="playlist",
+            field=models.ForeignKey(
+                help_text="playlist to which this classroom belongs",
+                on_delete=django.db.models.deletion.PROTECT,
+                related_name="%(class)ss",
+                to="core.playlist",
+                verbose_name="playlist",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="classroom",
+            name="position",
+            field=models.PositiveIntegerField(
+                default=0,
+                help_text="position of this classroom in the playlist",
+                verbose_name="position",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="classroom",
+            name="starting_at",
+            field=models.DateTimeField(
+                blank=True,
+                help_text="Start date and time at which a classroom is scheduled.",
+                null=True,
+                verbose_name="starting at date and time",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="classroom",
+            name="title",
+            field=models.CharField(
+                blank=True,
+                help_text="title of the classroom",
+                max_length=255,
+                null=True,
+                verbose_name="title",
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="classroom",
+            constraint=models.UniqueConstraint(
+                condition=models.Q(("deleted", None)),
+                fields=("lti_id", "playlist"),
+                name="classroom_unique_idx",
+            ),
+        ),
```

Comment:
I don't know how Django can create the constraint as the table `classroom` does not exist yet?

Reply:
This migration has been successfully applied on my side.